test/helpers/utils.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Utils to remove the _params key and all params keys from nested objects from
+ * an EasyPostObject for sake of comparison in tests
+ * @param {*} obj an EasyPostObject to remove the params from
+ * @returns {*} obj without the _params key
+ */
+export const withoutParams = (obj) =>
+  Object.fromEntries(
+    Object.entries(obj)
+      .map(([key, value]) =>
+        key === '_params'
+          ? null
+          : !value
+          ? [key, value]
+          : Array.isArray(value)
+          ? [
+              key,
+              value.map((arrValue) =>
+                typeof arrValue === 'object' ? withoutParams(arrValue) : arrValue,
+              ),
+            ]
+          : typeof value === 'object'
+          ? [key, withoutParams(value)]
+          : [key, value],
```

Comment:
This function is very hard to read, can we split them into multiple conditional branches?

Reply:
Fixed